src/templates/apps/party-sheet.html

```diff
@@ -2,19 +2,22 @@
   <div class="actor header flexrow item-controls">
     <div id="empty"></div>
     {{#if user.isGM}}
-    <button id="deal-xp" class="item-control" type="button" title="{{localize 'OSE.dialog.xp.deal'}}">
+    <button
+      id="deal-xp"
+      class="item-control"
+      type="button"
+      title="{{localize 'OSE.dialog.xp.deal'}}"
+    >
       {{localize 'OSE.dialog.xp.label'}}
     </button>
     {{/if}}
   </div>
   <div class="body party-members">
-    {{#if user.isGM}}
-    {{#if (eq partyActors.length 0)}}
+    {{#if user.isGM}} {{#if (eq partyActors.length 0)}}
```

Comment:
Rather than nesting ifs, could we add a prop to the backing JS file for this?

Reply:
Hope you'll accept this solution to eliminate nesting ifs
The `and` helper is from `foundry.js`